zjit/src/cruby.rs

```diff
@@ -245,14 +232,6 @@ pub fn insn_len(opcode: usize) -> u32 {
     }
 }
 
-/// Opaque iseq type for opaque iseq pointers from vm_core.h
-/// See: <https://doc.rust-lang.org/nomicon/ffi.html#representing-opaque-structs>
-#[repr(C)]
-pub struct rb_iseq_t {
```

Comment:
Nice. I wonder if this will resolve the issue I was running into trying to work with iseq* from within a debugger.